src/components/ContactCard/ContactForm.spec.jsx

```diff
@@ -1,28 +1,102 @@
 import React from 'react'
 import renderer from 'react-test-renderer'
-import { mount } from 'enzyme'
-import ContactForm from './ContactForm'
+import { mount, shallow } from 'enzyme'
+import ContactForm, { DumbContactForm, moveToHead } from './ContactForm'
 import { I18n } from 'cozy-ui/transpiled/react/I18n'
 
+import langEn from '../../locales/en.json'
+
+const dictRequire = () => langEn
+
+const contact = {
+  id: '9ecfbf4b-20e7-4bac-87f1-eea53350857d',
+  _id: '9ecfbf4b-20e7-4bac-87f1-eea53350857d',
+  _type: 'io.cozy.contacts',
+  _rev: '1-19c313536e8b27473aa26bf105b03269',
+  address: [
+    {
+      formattedAddress: '94 Hinton Road 05034 Fresno, Singapore',
+      type: 'Home',
+      primary: true
+    }
+    // TODO: add a non formatted address
+  ],
+  email: [
+    {
+      address: 'john.doe@posteo.net',
+      type: 'personal',
+      primary: false
+    },
+    {
+      address: 'john.doe@cozycloud.cc',
+      primary: true
+    }
+  ],
+  birthday: '1999-5-1',
+  company: 'Cozy cloud',
+  cozy: [
+    {
+      label: 'MyCozy',
+      primary: true,
+      url: 'https://johndoe.mycozy.cloud'
+    }
+  ],
+  fullName: 'John Doe',
+  name: {
+    givenName: 'John',
+    familyName: 'Doe'
+  },
+  metadata: {
+    version: 1
+  },
+  note:
+    'Atque cupiditate saepe omnis quos ut molestiae labore voluptates omnis.',
+  phone: [
+    {
+      number: '+33 (2)0 90 00 54 04',
+      primary: true
+    },
+    {
+      number: '+33 6 77 11 22 33',
+      primary: false
+    }
+  ]
+}
+
```

Comment:
I know this is not the scope of this PR. But we should create a fixture file for our spec ?

Reply:
you mean test data that can be used in various unit tests?

Reply:
yep

Reply:
I can move the contact I created in `src/helpers/testData.js` or somewhere else. It would be a starting point